main/guides/orchestration/how-orch-works.md

```diff
@@ -0,0 +1,38 @@
+# How Orchestration Works
+Blockchain orchestration fundamentally relies on protocols and mechanisms that enable blockchains to communicate and transact with each other securely and efficiently. The primary goals of blockchain orchestration are:
+
+- **Interoperability:** Allowing different blockchain networks to interact and transact with each other.
+- **Scalability:** Enabling multiple blockchains to work together without compromising performance.
+- **Security:** Ensuring that cross-chain transactions are secure and trustworthy.
+
+To achieve these goals, several foundational technologies and protocols have been developed, with the [Inter-Blockchain Communication (IBC) protocol](https://ibcprotocol.org/) being one of the most prominent.
+
+### Inter-Blockchain Communication (IBC)
+
+The [Inter-Blockchain Communication (IBC) protocol](https://ibcprotocol.org/) can be thought of as the TCP/IP of the blockchain world. Just like TCP/IP allows different computer networks to communicate over the internet, IBC enables different blockchains to communicate with each other.
+
+#### How IBC Works
+
+IBC operates on a simple premise: by adhering to a common set of communication protocols, blockchains can securely transfer data and tokens between one another. Here's a high-level overview of how IBC achieves this:
+
+- **Light Client Verification:** Each blockchain maintains a light client of the other blockchain. A light client is a simplified version of a blockchain client that only retains enough information to verify transactions and proofs, without needing to store the entire blockchain history.
+- **Relayer:** A [relayer](https://medium.com/@cosmos/relayer-in-blockchain-ecosystem-5f8b5b9b8b5e) is an off-chain process or entity that listens for transactions on one blockchain and relays them to another. The relayer facilitates communication between blockchains but does not have any special permissions or abilities; it merely passes messages between chains.
```

Comment:
The medium link isn’t working for me. I suggest using links from the Cosmos Network website or GitHub, like this one: https://github.com/cosmos/ibc/blob/main/spec/relayer/ics-018-relayer-algorithms/README.md. GitHub links are more reliable and likely to last longer since the platform is free and widely used.